src/features/common/components/version-display.test.tsx

```diff
@@ -99,7 +99,7 @@ describe('VersionDisplay', () => {
 
       expect(component.getByText(/Built:/)).toBeInTheDocument()
       expect(component.getByText(/Commit:/)).toBeInTheDocument()
-      expect(component.getByText(/1\/15\/2024/)).toBeInTheDocument() // Date formatting
+      expect(component.getByText(RegExp(`${new Date('2024-01-15').toLocaleDateString()}`))).toBeInTheDocument() // Date formatting
```

Comment:
This is so the test is locale agnostic. It failed on my local machine due to Australia format being 15/1/2025